### PR TITLE
Add SerialConvort to testers

### PR DESCRIPTION
**Pull request by SerialConvort**
Please add me to testers :v

### CODE REVIEW

File: src/data/developers.json [hunk 1]
```diff
@@ -330,6 +330,15 @@
 				"gh" :"https://github.com/RredderR"
 			}
 		}
```
Comment (dragonjet) [inline src/data/developers.json]: need `,` here :v

**Comment by ethrundr** [comment]
not was I against of adding new testers, but I haven't see any comment or issues that you've created before.

in my personal opinion, credit as Testers should only be granted once the person has contributed at least one using the Dev build.

**Comment by SerialConvort** [comment]
I just started now, planning to submit any kind of errors to DJ as soon as possible.

**Comment by McDohl** [comment]
sereal is here :open_mouth:
anyway welcome

**Pull request merged**